### PR TITLE
Update payment method names to align with brand guidelines

### PR DESCRIPTION
**Pull request by benycodes**
Fixed the capitalization in-line with the payment network's guidelines.

### CODE REVIEW

File: content/support/account-management-billing/Billing Cloudflare Plans/Cloudflare Billing Policy.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ ___
 
 ## Approved payment methods
 
-Cloudflare only accepts VISA, MasterCard, American Express, Discover, Paypal, and UnionPay. No other payment methods (for example, Maestro) are possible at this time.
```
Comment (MEGADIGITALSOLUTION) [inline content/support/account-management-billing/Billing Cloudflare Plans/Cloudflare Billing Policy.md:91]: Debit cards

**Review by kodster28 — APPROVED** [review]

**Review by MEGADIGITALSOLUTION — COMMENTED** [review]
Approve

**Review by MEGADIGITALSOLUTION — COMMENTED** [review]
Reviewed and approved

**Review by MEGADIGITALSOLUTION — COMMENTED** [review]
Approved @elithrar

**Review by MEGADIGITALSOLUTION — COMMENTED** [review]
Approve

File: content/support/account-management-billing/Billing Cloudflare Plans/Cloudflare Billing Policy.md (hunk 1 above)
Comment (MEGADIGITALSOLUTION) [reply content/support/account-management-billing/Billing Cloudflare Plans/Cloudflare Billing Policy.md:91]: Approve

**Pull request merged**